webpack.config.js

```diff
@@ -268,7 +274,8 @@ var legacyCssConfig = {
         loader: 'file-loader?name=' + '[name].[ext]'
       }
     ]
-  }
+  },
+  devtool: '#source-map'
```

Comment:
For JS, we use cheap source maps in dev for performance reasons, I think it's fine to always use `source-map` for css if the performance is negligible

Reply:
I wasn't even able to get `cheap-source-map` to work here